modules/special-rules/pages/war-engine-shields.adoc

```diff
@@ -23,11 +23,13 @@ When all of the unit's fields/shields are collapsed then any further hits will c
 You can track remaining fields/shields in a similar way to Damage Capacity -- just be sure to differentiate so that you are clear about which tracker relates to which feature on which War Engine.
 ====
 
-The main difference between the two technologies is that on collapse, power fields are destroyed and of no further use in the battle, while void shields may recover during the battle to provide protection again.
-You check to see if void shields recover at the appropriate step in the Rally phase.
+The main difference between the two technologies is that on collapse, power fields are destroyed and of no further use in the battle, while void shields may be repaired during the battle to provide protection again.
+You check to see if void shields are repaired at the appropriate step in the Rally phase.
+// IJW editing note: I've changed all 'recover' references to 'repair' for consistency with other pages.
 
 Neither power fields nor void shields offer any protection from close combat hits, as the enemy is inside the protective envelope at that point.
 {blank}[TODO: " -- though they do protect in a firefight" perhaps?]
+// IJW reply: I don't think the firefight text is needed, as this is effectively a duplicate of the CC page text, and there is no equivalent text on the firefight page.
```

Comment:
Okay, it was just a thought to mitigate against the possibility that some people might misconstrue from the text here in the broader association of assaults — but this might be overthinking it. I'm happy to leave it. I'll just queue editing notes for removal.
```suggestion
```